EIPS/eip-1679.md

```diff
@@ -28,6 +28,7 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
 
 ### Proposed EIPs
 
+- [EIP-131](https://github.com/ethereum/EIPs/pull/2024) : Add Precompiled BLAKE2b Contract
```

Comment:
This should be a link to https://eips.ethereum.org/EIPS/eip-131. Also please avoid the space before the colon.

Reply:
Added both